README.md

```diff
@@ -2398,8 +2400,85 @@ hazelcastClient.getMap('brothersMap').then(function (mp) {
 });
 ```
 
-## 7.9. Monitoring and Logging
+## 7.8. Performance
+
```

Comment:
I think we need to mention the "performance" here, since when I read the following, I don't see an explicit explanation on how PartitionAware affects the performance. A content explaining this would be useful, either here or within the section 7.8.1 below.